vdr/didnuts/creator.go

```diff
@@ -77,13 +77,31 @@ type Creator struct {
 // DefaultCreationOptions returns the default DIDCreationOptions when creating DID Documents.
 func DefaultCreationOptions() management.DIDCreationOptions {
 	return management.DIDCreationOptions{
-		Method:      MethodName,
-		Controllers: []did.DID{},
-		KeyFlags:    management.AssertionMethodUsage | management.CapabilityInvocationUsage | management.KeyAgreementUsage,
-		SelfControl: true,
+		Method:                MethodName,
+		KeyFlags:              management.AssertionMethodUsage | management.CapabilityInvocationUsage | management.KeyAgreementUsage,
+		MethodSpecificOptions: []management.DIDCreationOption{SelfControl(true)},
```

Comment:
if the API receives `SelfControl`=false, the `DIDCreateOptions` will contain 2 conflicting values. It works because it applies the last one, but it is complicated to understand. An options map is clearer, or the documentation on the usage of `DIDCreateOptions`/`MethodSpecificOptions` should be very clear.

Reply:
Fixed this, last option prevales